packages/sdk/src/types/TransactionExpiry.ts

```diff
@@ -28,9 +27,30 @@ class TransactionExpiry {
         public readonly expiryEpochSeconds: bigint
     ) {}
 
-    public toJSON(): number {
-        return Number(this.expiryEpochSeconds);
+    /**
+     * Get a string representation of the transaction expiry date in seconds since the Unix epoch.
+     * @returns {string} The string representation.
+     */
+    public toString(): string {
+        return this.expiryEpochSeconds.toString();
```

Comment:
**Suggestion:** I think it would be more useful for the user to see a date-time instead

Reply:
Right, I see the issue